microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/MicroProfileCompletions.java

```diff
@@ -96,6 +96,11 @@ public CompletionList doComplete(PropertiesModel document, Position position, Mi
 			// no completions
 			break;
 		case ASSIGN:
+			// Only collect if on right side of =
+			if (offset == node.getEnd()) {

```

Comment:
I think we probably want `offset >= node.getEnd()` because if the user has `microprofile.tools.formatting.surroundEqualsWithSpaces` on, they will probably trigger completion after a whitespace character `quarkus.http.cors = |`

![image](https://user-images.githubusercontent.com/20326645/88337236-5e11a780-cd04-11ea-8fe5-50eb41ebbd2c.png)

MP settings don't work on master at the moment though, we need https://github.com/eclipse/lsp4mp/pull/4

Reply:
Good catch. I added `>=`, and added this test case.